python.html.markdown

```diff
@@ -577,6 +577,8 @@ def double_numbers(iterable):
 # Instead of generating and returning all values at once it creates one in each
 # iteration.  This means values bigger than 15 wont be processed in
 # double_numbers.
+# Also, it means that the entire list won't be stored in memory.
```

Comment:
Generators are not lists. Frankly, I think the lines before and after already explains the concept pretty well.